app/assets/stylesheets/application.scss

```diff
@@ -237,3 +232,78 @@
   margin-top:100px;
   width:350px;
 }
+.sen-info{
+  margin-left:225px;
+  margin-top:25px;
+}
+body{
+  background-image:url('https://mk0dcphotoguide69e7p.kinstacdn.com/wp-content/uploads/2017/04/us-capitol-building-at-night-panorama-10-copyright-havecamerawilltravel-com.jpg');
+}
+// <style type='text/css' >body{background-color:rgb(0,0,0);background-image:url(https://mk0dcphotoguide69e7p.kinstacdn.com/wp-content/uploads/2017/04/us-capitol-building-at-night-panorama-10-copyright-havecamerawilltravel-com.jpg);background-position:top right;background-repeat:no-repeat;background-attachment:local;}</style>
+// style="margin-left:225px;margin-top:25px;"
+@media only screen and (min-width:1200px) and (max-width: 1600px) {
+  .sidenav {padding-top: 15px;}
+  .sidenav a {font-size: 18px;}
+  .fixed-image{
+    position:fixed;
+    margin-left:650px;
+    margin-top:100px;
+    width:350px;
+  }
+  .right-side-nav{
+    margin-left:78%;
+    margin-top:27px;
+  }
+  .sen-info{
+    margin-left:225px;
+    margin-top:-2em;
+  }
+  .bills-nav{
+    margin-right:160px;
+    font-size:20px;
+  }
+  .rep-info{
+    margin-left:225px;
+    margin-top:-2em;
+  }
+  body{
+    background-color:rgb(0,0,0);
+    background-size:132% 180%;
+    background-position: 80% 0%;
+    background-repeat:no-repeat;
+    background-attachment:local;
+  }
+  #submit-button{
+    margin-bottom:10px;
+  }
+  .dashboard{
+    height:500px;
+    width:900px;
+    margin-left:220px;
+  }
+  .compared-rep{
+    margin-left:59px;
+  }
+}
+@media only screen and (min-width:1600px){
+  body{
+    background-color:rgb(0,0,0);
+    background-size:132% 207%;
+    background-position: 80% 0%;
+    background-repeat:no-repeat;
+    background-attachment:local;
+  }
+  .sen-info{
+    margin-left:225px;
+  }
+  .rep-info{
+    margin-left:225px;
+  }
+  .compared-rep{
+    margin-left:-.5em;
+  }
+  .bills-nav{
+    margin-right:400px;
+    font-size:20px;
+  }
+}
```

Comment:
Interesting! Don't think I've ever said these words before, but CSS for the win!